### PR TITLE
Pass the profile setting default  schema for schema-related tests.

### PR DESCRIPTION
**Pull request by case-k-git**
Resolves #

### Description
I revised the configuration files to pass schema information, enabling integration testing using schemas

tested that we can create the table into destination schema

```
.tox/integration-databricks-uc-sql-endpoint/bin/python -m pytest -v -m profile_databricks_uc_sql_endpoint -n4 tests/integration/incremental_strategies/test_incremental_strategies.py::TestDeltaStrategiesWarehous
```

```
09:23:45  1 of 6 START sql incremental model schema_demo.append_delta .................... [RUN]
09:23:48  1 of 6 OK created sql incremental model schema_demo.append_delta ............... [OK in 2.40s]
09:23:48  2 of 6 START sql incremental model schema_demo.merge_exclude_columns ........... [RUN]
09:23:51  2 of 6 OK created sql incremental model schema_demo.merge_exclude_columns ...... [OK in 3.48s]
09:23:51  3 of 6 START sql incremental model schema_demo.merge_no_key .................... [RUN]
09:23:55  3 of 6 OK created sql incremental model schema_demo.merge_no_key ............... [OK in 3.51s]
09:23:55  4 of 6 START sql incremental model schema_demo.merge_unique_key ................ [RUN]
09:23:58  4 of 6 OK created sql incremental model schema_demo.merge_unique_key ........... [OK in 3.65s]
09:23:58  5 of 6 START sql incremental model schema_demo.merge_update_columns ............ [RUN]
09:24:02  5 of 6 OK created sql incremental model schema_demo.merge_update_columns ....... [OK in 3.67s]
09:24:02  6 of 6 START sql incremental model schema_demo.replace_where ................... [RUN]
09:24:05  6 of 6 OK created sql incremental model schema_demo.replace_where .............. [OK in 2.73s]
09:24:06  
09:24:06  Finished running 6 incremental models in 0 hours 0 minutes and 24.77 seconds (24.77s).
09:24:06  
09:24:06  Completed successfully
09:24:06  
09:24:06  Done. PASS=6 WARN=0 ERROR=0 SKIP=0 TOTAL=6
```

### Checklist

- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-databricks next" section.

### CODE REVIEW

**Comment by benc-db** [comment]
With this change, running with tox puts all of the tests into the same schema?  That would make cleanup much easier, assuming we don't have inadvertent collisions.  I'll migrate this to the main fork to see if the existing tests pass; if so, I think we're fine to merge this in.

**Comment by benc-db** [comment]
Going to re-open this on the main fork so that we can run integration tests.

File: tests/profiles.py [hunk 1]
```diff
@@ -32,6 +33,8 @@ def _build_databricks_cluster_target(
     }
     if catalog is not None:
         profile["catalog"] = catalog
+    if schema is not None:
+        profile["schema"] = schema
```
Comment (rcypher-databricks) [inline tests/profiles.py:37]: Even if the schema is set here doesn't it just get overwritten with the result of the unique_schema fixture in the dbt_profile_data fixture? That's what happens with I try this.

**Comment by case-k-git** [comment]
Thank you  for your checking. Yea sorry I confirm because of this change the exiting test failed. Failed count is change from  3 to 19 in my local env. I will check existing test.

File: tests/profiles.py (hunk 1 above)
Comment (case-k-git) [reply tests/profiles.py:37]: Thank you! I overwrite the unique_schema and some test is passed . But several test is faield

https://github.com/databricks/dbt-databricks/pull/538/commits/c4b9f8bcfc76b580798fcfcc7e20fd08d63c137c

```
.tox/integration-databricks-uc-sql-endpoint/bin/python -m pytest -v -m profile_databricks_uc_sql_endpoint -n4 tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
====================================================== test session starts =======================================================
platform darwin -- Python 3.11.5, pytest-7.4.3, pluggy-1.3.0 -- /Users/local_path/Desktop/workspace/dbt-databricks/.tox/integration-databricks-uc-sql-endpoint/bin/python
cachedir: .pytest_cache
rootdir: /Users/local_path/Desktop/workspace/dbt-databricks
configfile: pytest.ini
plugins: csv-3.0.0, flaky-3.7.0, dotenv-0.5.2, xdist-3.5.0
4 workers [1 item]      
scheduling tests via LoadScheduling

tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 
[gw1] [100%] PASSED tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 

======================================================== warnings summary ========================================================
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
  /Users/local_path/Desktop/workspace/dbt-databricks/dbt/adapters/databricks/connections.py:693: UserWarning: The cursor was closed by destructor.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================= 1 passed, 5 warnings in 8.62s ==================================================

```

**Comment by case-k-git** [comment]
@benc-db @rcypher-databricks 
I fix it could you please check this ?
https://github.com/databricks/dbt-databricks/pull/538/commits/14ca9407bb00ce4cf4de17add4f8cc7ded8cf5af

This change can pass schema from profile and will be easy to test it. Seems to be prefix is need it to make each dedicated test environment.
I also remove lower convert from unique  schema to test Schema Upper case. To investigate the issue that we talking the other PR
https://github.com/databricks/dbt-databricks/pull/530

Tested
```
% tox -e integration-databricks-uc-sql-endpoint
integration-databricks-uc-sql-endpoint: commands[0]> /bin/bash -c '/Users/user/Library/CloudStorage/OneDrive/Desktop/workspace/dbt-databricks/.tox/integration-databricks-uc-sql-endpoint/bin/python -m pytest -v -m profile_databricks_uc_sql_endpoint -n4 tests/integration/* ; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
=========================================== test session starts ===========================================
platform darwin -- Python 3.11.5, pytest-7.4.3, pluggy-1.3.0 -- /Users/user/Library/CloudStorage/OneDrive/Desktop/workspace/dbt-databricks/.tox/integration-databricks-uc-sql-endpoint/bin/python
cachedir: .tox/integration-databricks-uc-sql-endpoint/.pytest_cache
rootdir: /Users/user/Library/CloudStorage/OneDrive/Desktop/workspace/dbt-databricks
configfile: pytest.ini
plugins: csv-3.0.0, flaky-3.7.0, dotenv-0.5.2, xdist-3.5.0
4 workers [39 items]    
scheduling tests via LoadScheduling

tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_sync_all_columns 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_sync_all_columns 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_fail_on_schema_change 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_fail_on_schema_change 
[gw0] [  2%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_fail_on_schema_change 
[gw1] [  5%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_fail_on_schema_change 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_ignore 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_ignore 
[gw3] [  7%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_sync_all_columns 
tests/integration/incremental_strategies/test_incremental_strategies.py::TestDeltaStrategiesWarehouse::test_delta_strategies_databricks_uc_sql_endpoint 
[gw2] [ 10%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_sync_all_columns 
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_append_new_columns 
[gw0] [ 12%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaAppend::test__databricks_uc_sql_endpoint__run_incremental_ignore 
tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_base_databricks_uc_sql_endpoint 
[gw0] [ 15%] SKIPPED tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_base_databricks_uc_sql_endpoint 
tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_based_on_view_databricks_uc_sql_endpoint 
[gw0] [ 17%] SKIPPED tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_based_on_view_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestInvalidModelContractColumnConstraints::test_databricks_uc_sql_endpoint 
[gw1] [ 20%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_ignore 
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 
[gw1] [ 23%] PASSED tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 
tests/integration/liquid_clustering/test_liquid_clustering.py::TestLiquidClustering::test_liquid_clustering_databricks_uc_sql_endpoint 
[gw2] [ 25%] PASSED tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py::TestDeltaOnSchemaChange::test__databricks_uc_sql_endpoint__run_incremental_append_new_columns 
tests/integration/model_contract/test_model_contract.py::TestIncrementalModelContractConstraints::test_databricks_uc_sql_endpoint 
[gw0] [ 28%] PASSED tests/integration/model_contract/test_model_contract.py::TestInvalidModelContractColumnConstraints::test_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestTableWithModelContractConstraintsDisabled::test_databricks_uc_sql_endpoint 
[gw1] [ 30%] PASSED tests/integration/liquid_clustering/test_liquid_clustering.py::TestLiquidClustering::test_liquid_clustering_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestModelLevelPrimaryKey::test_databricks_uc_sql_endpoint 
[gw1] [ 33%] PASSED tests/integration/model_contract/test_model_contract.py::TestModelLevelPrimaryKey::test_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestModelContractNotDelta::test_databricks_uc_sql_endpoint 
[gw1] [ 35%] SKIPPED tests/integration/model_contract/test_model_contract.py::TestModelContractNotDelta::test_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestModelContractView::test_databricks_uc_sql_endpoint 
[gw1] [ 38%] PASSED tests/integration/model_contract/test_model_contract.py::TestModelContractView::test_databricks_uc_sql_endpoint 
tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogTableModels::test_multi_catalog_run_databricks_uc_sql_endpoint 
[gw1] [ 41%] SKIPPED tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogTableModels::test_multi_catalog_run_databricks_uc_sql_endpoint 
tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogViewModels::test_multi_catalog_run_databricks_uc_sql_endpoint 
[gw1] [ 43%] SKIPPED tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogViewModels::test_multi_catalog_run_databricks_uc_sql_endpoint 
tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogSeedsInAlternativeCatalog::test_multi_catalog_run_databricks_uc_sql_endpoint 
[gw1] [ 46%] SKIPPED tests/integration/multi_catalog/test_multi_catalog.py::TestMultiCatalogSeedsInAlternativeCatalog::test_multi_catalog_run_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestTableConstraints::test_databricks_uc_sql_endpoint 
[gw0] [ 48%] PASSED tests/integration/model_contract/test_model_contract.py::TestTableWithModelContractConstraintsDisabled::test_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestModelLevelForeignKey::test_databricks_uc_sql_endpoint 
[gw3] [ 51%] PASSED tests/integration/incremental_strategies/test_incremental_strategies.py::TestDeltaStrategiesWarehouse::test_delta_strategies_databricks_uc_sql_endpoint 
tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_no_cdf_databricks_uc_sql_endpoint 
[gw3] [ 53%] SKIPPED tests/integration/materialized_view/test_materialized_view.py::TestMaterializedView::test_materialized_view_no_cdf_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestModelContractConstraints::test_databricks_uc_sql_endpoint 
[gw2] [ 56%] PASSED tests/integration/model_contract/test_model_contract.py::TestIncrementalModelContractConstraints::test_databricks_uc_sql_endpoint 
tests/integration/model_contract/test_model_contract.py::TestInvalidModelContractCheckConstraints::test_databricks_uc_sql_endpoint 
[gw2] [ 58%] PASSED tests/integration/model_contract/test_model_contract.py::TestInvalidModelContractCheckConstraints::test_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestInvalidColumnConstraints::test_databricks_uc_sql_endpoint 
[gw2] [ 61%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestInvalidColumnConstraints::test_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestTableWithConstraintsDisabled::test_databricks_uc_sql_endpoint 
[gw0] [ 64%] PASSED tests/integration/model_contract/test_model_contract.py::TestModelLevelForeignKey::test_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestSnapshotConstraints::test_databricks_uc_sql_endpoint 
[gw1] [ 66%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestTableConstraints::test_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestIncrementalConstraints::test_databricks_uc_sql_endpoint 
[gw3] [ 69%] PASSED tests/integration/model_contract/test_model_contract.py::TestModelContractConstraints::test_databricks_uc_sql_endpoint 
tests/integration/persist_constraints/test_persist_constraints.py::TestInvalidCheckConstraints::test_databricks_uc_sql_endpoint 
[gw2] [ 71%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestTableWithConstraintsDisabled::test_databricks_uc_sql_endpoint 
tests/integration/run_results_json/test_run_results_json.py::TestRunResultsJson::test_run_results_json_databricks_uc_sql_endpoint 
[gw3] [ 74%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestInvalidCheckConstraints::test_databricks_uc_sql_endpoint 
tests/integration/store_failures/test_store_failures.py::TestStoreFailuresDelta::test_store_failures_databricks_uc_sql_endpoint 
[gw2] [ 76%] PASSED tests/integration/run_results_json/test_run_results_json.py::TestRunResultsJson::test_run_results_json_databricks_uc_sql_endpoint 
tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_base_databricks_uc_sql_endpoint 
[gw2] [ 79%] SKIPPED tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_base_databricks_uc_sql_endpoint 
tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_no_cdf_databricks_uc_sql_endpoint 
[gw2] [ 82%] SKIPPED tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_no_cdf_databricks_uc_sql_endpoint 
tests/integration/zorder/test_zorder.py::TestZOrder::test_zorder_databricks_uc_sql_endpoint 
[gw0] [ 84%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestSnapshotConstraints::test_databricks_uc_sql_endpoint 
tests/integration/seed_column_types/test_seed_column_types.py::TestSeedColumnTypeCast::test_seed_column_types_databricks_uc_sql_endpoint 
[gw3] [ 87%] PASSED tests/integration/store_failures/test_store_failures.py::TestStoreFailuresDelta::test_store_failures_databricks_uc_sql_endpoint 
tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_based_on_view_databricks_uc_sql_endpoint 
[gw3] [ 89%] SKIPPED tests/integration/streaming_table/test_streaming_table.py::TestStreamingTable::test_streaming_table_based_on_view_databricks_uc_sql_endpoint 
[gw0] [ 92%] PASSED tests/integration/seed_column_types/test_seed_column_types.py::TestSeedColumnTypeCast::test_seed_column_types_databricks_uc_sql_endpoint 
[gw1] [ 94%] PASSED tests/integration/persist_constraints/test_persist_constraints.py::TestIncrementalConstraints::test_databricks_uc_sql_endpoint 
tests/integration/set_tblproperties/test_set_tblproperties.py::TestSetTblproperties::test_set_tblproperties_databricks_uc_sql_endpoint 
[gw2] [ 97%] PASSED tests/integration/zorder/test_zorder.py::TestZOrder::test_zorder_databricks_uc_sql_endpoint 
[gw1] [100%] PASSED tests/integration/set_tblproperties/test_set_tblproperties.py::TestSetTblproperties::test_set_tblproperties_databricks_uc_sql_endpoint 

============================================ warnings summary =============================================
tests/integration/incremental_on_schema_change/test_incremental_on_schema_change.py: 38 warnings
tests/integration/incremental_strategies/test_incremental_strategies.py: 16 warnings
tests/integration/model_contract/test_model_contract.py: 70 warnings
tests/integration/liquid_clustering/test_liquid_clustering.py: 5 warnings
tests/integration/persist_constraints/test_persist_constraints.py: 55 warnings
tests/integration/run_results_json/test_run_results_json.py: 5 warnings
tests/integration/store_failures/test_store_failures.py: 5 warnings
tests/integration/seed_column_types/test_seed_column_types.py: 5 warnings
tests/integration/zorder/test_zorder.py: 5 warnings
tests/integration/set_tblproperties/test_set_tblproperties.py: 11 warnings
  /Users/user/Library/CloudStorage/OneDrive/Desktop/workspace/dbt-databricks/dbt/adapters/databricks/connections.py:693: UserWarning: The cursor was closed by destructor.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 29 passed, 10 skipped, 215 warnings in 267.91s (0:04:27) =========================

```

File: tests/integration/base.py [hunk 2]
```diff
@@ -165,13 +165,12 @@ def packages_config(self):
     @property
     def selectors_config(self):
         return None
-
-    def unique_schema(self):
-        schema = self.schema
-
-        to_return = "{}_{}".format(self.prefix, schema)
-
-        return to_return.lower()
```
Comment (case-k-git) [inline tests/integration/base.py:174]: I removed lower convert to test schema upper case

Comment (case-k-git) [reply tests/integration/base.py:174]: I created the PR about this separately

https://github.com/databricks/dbt-databricks/pull/541

**Comment by benc-db** [comment]
When I run now, the SQL Warehouse and UC Cluster profiles have all tests passing, but for non-UC, the incremental tests fail.

**Comment by case-k-git** [comment]
Hello @benc-db Thank you for your review I fix it could you please check this as well?

https://github.com/databricks/dbt-databricks/pull/538/commits/2d59088130cb985adfa88314918a62314e8fe173

Initially I though it will be good to use common schema but there are use case that we want to test schema related one like this PR use case.
https://github.com/databricks/dbt-databricks/pull/530

So in this PR I can set up default schema. if schema is define in test code we can overwrite default schema, if not default schema will be apply. We will be able to remove schema from existing test and will be simple. we can only define when we want to test schema related one.

>When I run now, the SQL Warehouse and UC Cluster profiles have all tests passing, but for non-UC, the incremental tests fail.

I checked Github Actions log and fix it to check schema existence. Sorry I did not test it locally because of lack of my permission and might be difficult to get it in my company security related policy.

```
self = <test_zorder.TestZOrder testMethod=test_zorder_databricks_cluster>
adapter_type = 'databricks_cluster'

    def get_profile(self, adapter_type):
        profile = {
            "config": {"send_anonymous_usage_stats": False},
            "test": {
                "outputs": {
                    "dev": dict(
                        **get_databricks_cluster_target(adapter_type),
                    ),
                },
                "target": "dev",
            },
        }
>       profile["test"]["outputs"]["dev"]["schema"] = self.unique_schema(schema = profile["test"]["outputs"]["dev"]["schema"])
E       KeyError: 'schema'
```

**Comment by benc-db** [comment]
> I checked Github Actions log and fix it to check schema existence. Sorry I have checked UC but did not test non-UC cluster locally because of lack of my permission and might be difficult to get it in my company security related policy(if I can test by CI/CD it will be very helpful).

I agree that this is not ideal; unfortunately my company's policies prevent me from giving you direct access.  I'll run them again with latest code.

**Comment by benc-db** [comment]
Looks like this is at issue in the HMS tests: 
Parsing Error
  at path ['schema']: None is not of type 'string'

**Comment by case-k-git** [comment]
@benc-db 
Thank you for your check I fix it hope solve the issue. Set the default schema for non UC testing
https://github.com/databricks/dbt-databricks/pull/538/commits/de9e39578ce6f43a18dae27c798f0a5e5c2cfc5c

>I agree that this is not ideal; unfortunately my company's policies prevent me from giving you direct access. I'll run them again with latest code.

Yea I can understand. Non UC catalog will probably need `CREATE privilege on CATALOG` role. I do not have that permission so I test it locally like only skip schema part. Hope this will fix the issue

**Comment by case-k-git** [comment]
@benc-db
Thank you for your check I fix this issue hope solve the issue. Even non uc test schema is need it so I  set the default schema for non UC testing
https://github.com/databricks/dbt-databricks/pull/538/commits/853b579b9a78ed6eb339794bfba7b8c2d90d9c69

>I agree that this is not ideal; unfortunately my company's policies prevent me from giving you direct access. I'll run them again with latest code.

Yea I can understand. Non UC catalog will probably need `CREATE privilege on CATALOG` role.

**Comment by benc-db** [comment]
Looks like all of the integration tests are passing now.  Will open a new PR to finalize.

**Comment by benc-db** [comment]
https://github.com/databricks/dbt-databricks/pull/548/

**Pull request closed without merging**